include/Macro.h

```diff
@@ -84,6 +84,12 @@
 #define NUC_TABLE_MODE_ENGY 2
 
 
+// nuclear eos solvers
+#define ORIG   1
+#define LUT    2
+#define DIRECT 3
```

Comment:
I agree moving them to `Nuclear.h` is better. One quick question, should I also move `NUC_TABLE_MODE_*` to `Nuclear.h`?

Reply:
I'm worried that some checks in `Macro.h` (e.g., `# if ( NUC_TABLE_MODE == NUC_TABLE_MODE_TEMP )`) *may* become incorrect since `NUC_TABLE_MODE_TEMP/ENGY` may have not been defined at this point. Can you check? If so, it may be safer and more consistent to define both `NUC_TABLE_MODE` and `NUC_EOS_SOLVER_*` in `Macro.h`.

I don't have a strong opinion here. What do you think?

Reply:
True, since `NUC_TABLE_MODE` is more widely used than `NUC_EOS_SOLVE`, and is everywhere. I think it is better to keep `NUC_TABLE_MODE` in `Macro.h`

Reply:
Then I suggest keeping both of them in `Macro.h` for consistency and safety.

Reply:
Got it.

Reply:
Add the prefix `NUC_EOS_SOLVER_` to `ORIG / LUT / DIRECT` in all places.